ush/hpssarch_gen.sh

```diff
@@ -584,5 +584,175 @@ if [ $type = "enkfgdas" -o $type = "enkfgfs" ]; then
+  done # loop over FHR
+  for fstep in ecenfsoi esfcfsoi eupdfsoi efcsfsoi eposfsoi ; do
+     echo  "logs/${CDATE}/${CDUMP}${fstep}*.log        " >>efsoi${CDUMP}.txt
+  done
+
+
+# Ensemble spread file only available with netcdf output
+  fh=6
+  while [ $fh -le 30 ]; do
+      fhr=$(printf %03i $fh)
+      echo  "${dirname}${head}atmf${fhr}.ensmean${SUFFIX}       " >>efsoi${CDUMP}.txt
+      echo  "${dirname}${head}sfcf${fhr}.ensmean${SUFFIX}       " >>efsoi${CDUMP}.txt
+      if [ $OUTPUT_FILE = "netcdf" ]; then
+          if [ -s $ROTDIR/${dirpath}${head}atmf${fhr}.ensspread${SUFFIX} ]; then
+	     echo  "${dirname}${head}atmf${fhr}.ensspread${SUFFIX}     " >>efsoi${CDUMP}.txt
+          fi
+      fi
+      fh=$((fh+6))
+  done
+
+  #...........................
+  n=1
+  while [ $n -le $NTARS ]; do
+     #...........................
+   
+     rm -f efsoi${CDUMP}_grp${n}.txt
+     rm -f efsoi${CDUMP}_restarta_grp${n}.txt
+     rm -f efsoi${CDUMP}_restartb_grp${n}.txt
+     touch efsoi${CDUMP}_grp${n}.txt
+     touch efsoi${CDUMP}_restarta_grp${n}.txt
+     touch efsoi${CDUMP}_restartb_grp${n}.txt
+   
+     m=1
+     while [ $m -le $NMEM_EARCGRP ]; do
+        nm=$(((n-1)*NMEM_EARCGRP+m))
+        mem=$(printf %03i $nm)
+        dirpath="efsoi${CDUMP}.${PDY}/${cyc}/atmos/mem${mem}/"
+        dirname="./${dirpath}"
+        head="${CDUMP}.t${cyc}z."
+    
+        #---
+        for FH in $nfhrs; do  # loop over analysis times in window
+           FHR=$(printf %03i $FH)
+           if [ $FHR -eq 6 ]; then
+              if [ $n -le $NTARS2 ]; then
+                 if [ -s $ROTDIR/${dirpath}${head}atmanl${SUFFIX} ] ; then
+                     echo "${dirname}${head}atmanl${SUFFIX}      " >>efsoi${CDUMP}_grp${n}.txt
+                 fi
+     	    if [ -s $ROTDIR/${dirpath}${head}ratminc${SUFFIX} ] ; then
+     		echo "${dirname}${head}ratminc${SUFFIX}      " >>efsoi${CDUMP}_grp${n}.txt
+     	    fi
+              fi
+              if [ -s $ROTDIR/${dirpath}${head}ratminc${SUFFIX} ] ; then
+                  echo "${dirname}${head}ratminc${SUFFIX}      " >>efsoi${CDUMP}_restarta_grp${n}.txt
+              fi
+     
+           else
+              if [ $n -le $NTARS2 ]; then
+                  if [ -s $ROTDIR/${dirpath}${head}atma${FHR}${SUFFIX} ] ; then
+                      echo "${dirname}${head}atma${FHR}${SUFFIX}      " >>efsoi${CDUMP}_grp${n}.txt
+                  fi
+                  if [ -s $ROTDIR/${dirpath}${head}ratmi${FHR}${SUFFIX} ] ; then
+                      echo "${dirname}${head}ratmi${FHR}${SUFFIX}      " >>efsoi${CDUMP}_grp${n}.txt
+                  fi
+              fi
+              if [ -s $ROTDIR/${dirpath}${head}ratmi${FHR}${SUFFIX} ] ; then
+                  echo "${dirname}${head}ratmi${FHR}${SUFFIX}      " >>efsoi${CDUMP}_restarta_grp${n}.txt
+              fi
+     
+           fi 
+           echo "${dirname}${head}atmf${FHR}${SUFFIX}       " >>efsoi${CDUMP}_grp${n}.txt
+           if [ $FHR -eq 6 ]; then
```

Comment:
Same

Reply:
Changing to reflect the scheme used by regular enkf in same file